templates/shared/pricing/_kubernetes-enterprise-support.html

```diff
@@ -1,99 +1,109 @@
-  </tr>
-  <tr>
-    <th>Livepatch</th>
-    <td>&nbsp;</td>
-    <td class="p-table__cell--highlight u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
-    <td class="p-table__cell--highlight u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
-    <td class="p-table__cell--highlight u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
-  </tr>
-  <tr>
-    <th>Knowledge Base</th>
-    <td>&nbsp;</td>
-    <td class="p-table__cell--highlight u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
-    <td class="p-table__cell--highlight u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
-    <td class="p-table__cell--highlight u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
-  </tr>
-  <tr>
-    <th>High availability support</th>
-    <td>&nbsp;</td>
-    <td class="p-table__cell--highlight">&nbsp;</td>
-    <td class="p-table__cell--highlight u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
-    <td class="p-table__cell--highlight u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
-  </tr>
-  <tr>
-    <th>24x7 remote operations, smart alerts and proactive monitoring by Canonical’s cloud experts</th>
-    <td>&nbsp;</td>
-    <td class="p-table__cell--highlight">&nbsp;</td>
-    <td class="p-table__cell--highlight">&nbsp;</td>
-    <td class="p-table__cell--highlight u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
-  </tr>
-  <tr>
-    <th>Disaster recovery</th>
-    <td>&nbsp;</td>
-    <td class="p-table__cell--highlight">&nbsp;</td>
-    <td class="p-table__cell--highlight">&nbsp;</td>
-    <td class="p-table__cell--highlight u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" alt="Yes"></td>
-  </tr>
-</table>
+<div class="row">
+  <div class="col-12" style="overflow-x: auto;">
+    <table style="width: 100%;">
+      <tr>
+        <td width="35%">&nbsp;</td>
+        <th width="15%" class="u-align--center">Ubuntu</th>
+        <th class="p-table__cell--highlight u-align--center" colspan="2">Ubuntu Advantage for Kubernetes</th>
+        <th class="p-table__cell--highlight u-align--center">BootStack</th>
+      </tr>
+      <tr>
+        <td>&nbsp;</td>
+        <td>&nbsp;</td>
+        <th class="p-table__cell--highlight u-align--center">Standard</th>
+        <th class="p-table__cell--highlight u-align--center">Advanced</th>
+        <th class="p-table__cell--highlight u-align--center">Fully Managed</th>
+      </tr>
+      <tr>
+        <th>Price per node (physical) <span style="font-size: 1rem;">*</span></th>
```

Comment:
These are `1rem` already. You don't need the style attribute. Shouldn't this link to the foot note?